src/controller/c/supervisor.c

```diff
@@ -2055,6 +2083,61 @@ const char *wb_supervisor_node_get_base_type_name(WbNodeRef node) {
 
   return wb_node_get_name(node->type);
 }
+WbFieldRef wb_supervisor_node_get_field_by_index(WbNodeRef node, const int id) {
```

Comment:
Argument name should match the one in `supervisor.h`
```suggestion
WbFieldRef wb_supervisor_node_get_field_by_index(WbNodeRef node, int index) {
```

same in `wb_supervisor_node_get_proto_field_by_index`.